README.md

```diff
@@ -113,6 +136,10 @@ Anyone and everyone is welcome to [contribute](https://github.com/Esri/maps-app-
 3. Contribute code
 4. Improve documentation
 
+## MDTOC
+
+Generation a table of contents in this repository was performed using the [MDTOC package for Atom](https://atom.io/packages/atom-mdtoc).
```

Comment:
I tend to prefer the original wording since it clarifies where these TOCs can be found:

```suggestion
Generation of this and other documents' table of contents in this repository was performed using the [MDTOC package for Atom](https://atom.io/packages/atom-mdtoc).
```

or if we want to simplify it:

```suggestion
Generating table of contents for documents in this repository was performed using the [MDTOC package for Atom](https://atom.io/packages/atom-mdtoc).
```